rust/ostree-host/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ostree-host"
+version = "0.1.0"
+authors = ["Colin Walters <walters@verbum.org>", "Jonathan Lebon <jonathan@jlebon.com>"]
```

Comment:
Feel free to drop my authorship, I don't think I've written much to `ostree_utils.rs`.

Reply:
I am looking forward to https://rust-lang.github.io/rfcs/3052-optional-authors-field.html